src/System.Net.Sockets/tests/FunctionalTests/UdpClientTest.cs

```diff
@@ -688,6 +688,19 @@ public void UdpReceiveResult_Equality()
             Assert.True(new UdpReceiveResult(buffer1, ep1) != new UdpReceiveResult(buffer1, ep3));
         }
 
+        [Fact]
+        public void BeginSend_IPv6Socket_IPv4Dns_Success()
+        {
+            using (var sender = new UdpClient(AddressFamily.InterNetworkV6))
+            {
+                sender.Client.DualMode = true;
+                for (int i = 0; i < TestSettings.UDPRedundancy; i++)
+                {
+                    sender.Send(new byte[1], 1, "114.114.114.114", 53);
```

Comment:
it seems like this is actual name server. I don't think we should send there malformed data.
It seems like one of the previous iteration had "loopback" here.  If that works, you can use "127.0.0.1". 
Since we do not care if packet is delivered, I don't think we need loop here.
I would also recommend use "discard" port (9) to avoid any unexpected interference.  
I think we should be ok as long as this does not throw name failure exception, right?